### ISSUE TITLE
Remove Guzzle and php-http/message

### ISSUE THREAD

**Issue by Nyholm** [graphaware/neo4j-php-client#107]
This is a BC break but it should be done before releasing next major version. 
This issue is just a reminder. 

Remove these lines from composer.json
```json
"php-http/message": "^1.0",
"php-http/guzzle6-adapter": "^1.0"
```

We should require the virtual package `php-http/client-implementation`

**Comment by ikwattro** [comment]
Hi @Nyholm . I'm still undecided about what the next release will be. I had a call with the neo technology driver team last week and for sure there will be no improvements regarding http on the neo4j side and it might be completely removed in neo4j 4.x. 

So I'm not sure v5 of the client will support http at all.

**Comment by Nyholm** [comment]
True. I forgot about that. Good that we have it documented here. 

I've no intentions in getting this merged now (or ever), but I wanted it to have as a reminder. 

Btw, I may see a future where we remove the HTTP things to a separate package (just like the bolt one)

**Comment by ruudk** [comment]
Any news on this?
